includes/admin/class-coblocks-crop-settings.php

```diff
@@ -92,6 +114,7 @@ public function get_original_image() {
 	 * Cropping.
 	 */
 	public function api_crop() {
+		wp_verify_nonce( $_POST['cropSettingsNonce'], 'cropSettingsNonce' );
```

Comment:
We'll want to sanitize the nonce value before using it.

```php
$nonce = filter_input( INPUT_POST, 'cropSettingsNonce', FILTER_SANITIZE_STRING );

if ( ! $nonce ) {

   return;

}

wp_verify_nonce( $nonce, 'cropSettingsNonce' );
...
```

Reply:
Fixed.